avery_website/assets/sass/all.sass

```diff
@@ -79,8 +79,10 @@ h6
     background-color: $color-bg
     color: $color-primary
 
-    .container
+    .container-top
```

Comment:
Can you include a comment in here that explains why these were split?

Reply:
Was response to adding to heading, since reverted by [4e368c1](https://github.com/averyhouse/avery-website/pull/3/commits/4e368c13f5a0e9ca80b8320a2727d71eb7585561)
no longer needed.